packages/block-library/src/image/style.scss

```diff
@@ -7,8 +7,11 @@
 		vertical-align: bottom;
 	}
 
-	&:not(.is-style-rounded) img {
-		border-radius: inherit;
+	&:not(.is-style-rounded) {
+		a,
```

Comment:
I assume it doesn't really matter (I don't think border-radius would really affect things), but will this `a` rule also target the `a` tags within a caption where a user has added an inline link inside the RichText caption field? If it is an issue, would it be worth adding a child combinator to this rule (`>`)?

Reply:
> but will this a rule also target the a tags within a caption

Good point - have narrowed down the scope to direct children.

Reply:
I retested the update. LGTM. 

Shame there's not a way to target the `img` tag directly, and "skip" a generation in the CSS inheritance chain. I guess it's acting like real life in that way 😆